crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -71,6 +71,8 @@ pub fn dim_inference(node: &mut Node, graph_io: &mut OnnxGraphIO) {
         NodeType::PRelu => same_as_input(node),
         NodeType::Where => where_update_outputs(node),
         NodeType::Squeeze => squeeze_update_output(node),
+        NodeType::RandomUniform => explicit_shape_update_output(node),
+        NodeType::RandomNormal => explicit_shape_update_output(node),
```

Comment:
This PR only tackles `RandomUniform`, mind removing the `RandomNormal`? Just so the appropriate warning is emitted when trying to go through the random normal conversion on import.

Reply:
Since `RandomNormal` has no input, `temporary_pass_through_stub` crashes trying to copy the input's dimensions.
But since `RandomNormal` and `RandomUniform` are essentially the same, I'll just also implement `RandomNormal` as well.

Would you rather have two almost exactly copy-paste nodes, or a single node with a `Distribution`-Parameter? If the latter, any input on naming the struct? Just `RandomNode`?

Reply:
I don't mind having two almost exactly the same nodes, but `RandomDistributionNode` with a `Distribution` field sounds good if you want to take a stab at this.